webbpsf/tests/test_wfirst.py

```diff
@@ -108,3 +108,17 @@ def test_WFI_limits_interpolation_range():
     assert 'wavelength outside the range' in str(excinfo.value), (
         "FieldDependentAberration did not error on out-of-bounds wavelength"
     )
+
+def test_CGI_psf():
+    """
+    Just test that instantiating CGI works and can compute a PSF without raising
+    any exceptions
+    """
+    char_spc = wfirst.CGI()
+    char_spc.mode = 'CHARSPC'
+
+    #print('Reading instrument data from {:s}'.format(charspc._WebbPSF_basepath)
+    #print('Filter list: {:}'.format(charspc.filter_list))
+
+    monopsf = char_spc.calcPSF(nlambda=1, display=True)
```

Comment:
Let's think of some simple assertions we could add here to check properties of the output PSF at a basic level.  Pixel dimensions, total flux, contrast, whatever.  The test becomes much more comprehensive if it's actually looking at the pixel values of the output PSF.